README.md

```diff
@@ -136,6 +137,14 @@ spec:
    rangeEnd: "FD:FF:FF:FF:FF:FF"
 ```
 
+## Nmstate
+
+The operator allows the administrator to deploy the [NmState State Controller](https://github.com/nmstate/nmstate) as a daemonset across all of one's nodes.
+This project manages host networking settings in a declarative manner. The networking state is described by a pre-defined schema. Reporting of current state and changes to it (desired state) both conform to the schema.
+Nmstate is aimed to satisfy enterprise needs to manage host networking through a northbound declarative API and multi provider support on the southbound. NetworkManager acts as the main (and currently the only) provider supported.
+
+Future work will include managing Nmstate settings through the operator as well 
```

Comment:
Can you start with `**Note:** ` and add a point at the end please

Reply:
Or maybe remove the note? There is no need to describe future work here IMO, better open an issue to track the effort. Just an idea, won't block with this.